config/config.go

```diff
@@ -65,6 +66,7 @@ var (
 		CursorFlushPeriod:  5 * time.Second,
 		CursorSeekFallback: SeekPositionTail,
 		DefaultType:        "journal",
+		FlushLogInterval:   30 * time.Second,
```

Comment:
Already parametrized. This is the default in case the user is not specifying it in the yaml file.